pkg/controllers/chart/chart.go

```diff
@@ -377,6 +393,9 @@ func job(chart *v1.HelmChart, apiServerPort string) (*batch.Job, *corev1.Secret,
 		chartName = chart.Name + "/" + chart.Spec.Chart
 	}
 
+	podSecurityContext := deepCopyStruct(defaultPodSecurityContext).(*corev1.PodSecurityContext)
+	securityContext := deepCopyStruct(defaultSecurityContext).(*corev1.SecurityContext)
```

Comment:
Do we really need to set both? As the upstream docs note:
> PodSecurityContext holds pod-level security attributes and common container settings. Some fields are also present in container.securityContext. Field values of container.securityContext take precedence over field values of PodSecurityContext.

Is there something specific we're setting in the SecurityContext, that we couldn't get just by setting the PodSecurityContext?